snap/snapcraft.yaml

```diff
@@ -183,15 +183,18 @@ parts:
       # These are only included here until the OS snap stabilizes
       - libxml2
       - libpng12-0
+        #- libmagickwand-6.q16-2
+        #- libmagickcore-6.q16-2
```

Comment:
Were you able to verify that the snap can still use magick with these gone?

Reply:
No, but I'm compiling snaps with this code right now and let you know!